src/include/catalog/namespace.h

```diff
@@ -35,6 +35,7 @@ typedef struct _FuncCandidateList
 	int			nvargs;			/* number of args to become variadic array */
 	int			ndargs;			/* number of defaulted args */
 	int		   *argnumbers;		/* args' positional indexes, if named call */
+	List	   *argdefaults;    /* list of default args, only set for PL/tsql function */
```

Comment:
Using tsql-specific names, e.g., tsql_argdefaults, may help us to resolve conflicts with the community PG in the future.